src/styles/main.scss

```diff
@@ -0,0 +1,89 @@
+// Color variables
+$border-color: #000;
+$day-color: #eee;
+$hover-color: #ffbfcb;
+
+// Size variables
```

Comment:
Remove comments
Also you can create a separate file for variables and each BEM block